lesson_15/roman_alex/lesson_15_roman_alex.py

```diff
@@ -0,0 +1,34 @@
+from lesson_9_roman_alex.py import Client, Gold_client
+import unittest
+
+
+class Test_Gold_Client(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Nice job for finding those methods, this classes are used for preparing a general state for the test run, like connecting and populating a database and then at tearDown deleting that database